gui/RespiraConnectedDevice.h

```diff
@@ -0,0 +1,115 @@
+#include "../common/generated_libs/network_protocol/network_protocol.pb.h"
+#include "../common/third_party/nanopb/pb_common.h"
+#include "../common/third_party/nanopb/pb_decode.h"
+#include "../common/third_party/nanopb/pb_encode.h"
+#include "connected_device.h"
+#include <QSerialPort>
+#include <QThread>
+
+#define INTER_FRAME_TIMEOUT 11
+
+class RespiraConnectedDevice : public ConnectedDevice {
+
+public:
+  RespiraConnectedDevice(QString portName, uint32_t write_timeout_ms,
+                         uint32_t interframe_timeout_ms)
+      : serialPortName(portName), m_writeTimeout_ms(write_timeout_ms),
```

Comment:
Per previous comments, please name the private variables as some_private_var_ rather than m_privateVar